services/chaospod.go

```diff
@@ -187,6 +187,12 @@ func (m *chaosPodService) HandleChaosPodTermination(ctx context.Context, disrupt
 				continue
 			}
 
+			// if the injector container definitively did not start, we can remove the finalizer
+			if cs.Started != nil && !(*cs.Started) {
```

Comment:
cs.Started == nil should remove the finalizer no?

Reply:
I have never yet seen a circumstance (outside of unit tests) where that field is nil. Either the pod didn't have any container statuses yet, or this field was set. It's probable there's a tiny window where it's nil, that i just haven't seen. I think for now I'll err on assuming that when Started is nil, it's actually unknown if it started, rather than knowing it didn't start.